src/cheat_gui.cpp

```diff
@@ -43,7 +43,7 @@
  * This variable is semantically a constant value, but because the cheat
  * code requires to be able to write to the variable it is not constified.
  */
-static int32 _money_cheat_amount = 10000000;
+static Money _money_cheat_amount = 10000000;
```

Comment:
This change would require line 189 to change as well to SLE_INT64 (if that even works).

Reply:
No, it silently casts it to the right thing. But let's not make this PR more complicated than we have to ;) Fixed it by leaving it an `int32`